src/Components/CreateImageWizard/formComponents/RepositoriesStatus.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import {
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  ExclamationTriangleIcon,
+  InProgressIcon,
+} from '@patternfly/react-icons';
+import { Alert, Button, Popover } from '@patternfly/react-core';
+
+type RepositoryStatusProps = {
+  repoStatus: string;
+  repoUrl: string;
+};
+
+const RepositoriesStatus = ({ repoStatus, repoUrl }: RepositoryStatusProps) => {
+  if (repoStatus === 'Valid') {
+    return (
+      <>
+        <CheckCircleIcon className="success" /> {repoStatus}
+      </>
+    );
+  } else if (repoStatus === 'Invalid') {
+    return (
+      <>
+        <Popover
+          position="bottom"
+          bodyContent={
+            <>
+              <Alert
+                variant="danger"
+                title="Invalid"
+                isInline
+                isPlain
+              />
+              Cannot fetch {repoUrl}
+            </>
+          }
+        >
+          <Button variant="link" className="pf-u-p-0 pf-u-font-size-sm">
+            <div className="failure-button">
+              <ExclamationCircleIcon className="error" /> {repoStatus}
+            </div>
+          </Button>
+        </Popover>
+      </>
+    );
+  } else if (repoStatus === 'Unavailable') {
+    return (
+      <>
+        <Popover
+          position="bottom"
+          bodyContent={
+            <>
+              <Alert
+                variant="warning"
+                title="Unavailable"
+                isInline
+                isPlain
+              />
+              Cannot fetch {repoUrl}
+            </>
+          }
+        >
+          <Button variant="link" className="pf-u-p-0 pf-u-font-size-sm">
+            <div className="failure-button">
+              <ExclamationTriangleIcon className="expiring" /> {repoStatus}
+            </div>
+          </Button>
+        </Popover>
+      </>
+    );
+  } else {
```

Comment:
This is Javascript but I'm taking inspiration from PEP 20 - The Zen of Python, "Explicit is better than implicit". 

I think it would be more explicit to use an `else if` here where repoStatus === 'Pending', or whatever the exact value is.

The exact values don't seem to show up in our generated API types, which seems to be a bit of an oversight in their API spec... unless I'm missing something?

Reply:
Agree, but I'm also unable to find the exact value. Am I looking in the right place? The status seems to be just a string in their API spec.

Reply:
Ok se the last value would be probably "Pending"

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/ac79848b-4c88-4b39-ab59-b8dc692d6a7c)

Reply:
Updated